specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,45 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the KeyVault key."
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The version of the KeyVault key."
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "The URI of the KeyVault."
+        },
+        "accessCredentials": {
+          "$ref": "#/definitions/AzureActiveDirectoryApplicationCredentials",
+          "description": "The credentials to use to authenticate to the provided KeyVault. Can be left unspecified if MSI (Managed Service Identity) is to be used instead."
```

Comment:
Can be left unspecified if MSI (Managed Service Identity) is enabled for the service - see https://aka.ms/azure-search-msi